### PR TITLE
FIDO Authenticator specific components multi language support and bug fix.

### PR DESCRIPTION
**Pull request by AfraHussaindeen**
### Proposed changes in this pull request

- Make UI error message display consistent. [1]
- Fix incorrect redirection upon user initiated authentication cancel request
- Add multi language support & minor existing content fixing.
- Add the error message content that will be displayed upon provisioned user not found.
- Fix myAccount redirection URL issue
- Enable language switch
- Relates to https://github.com/wso2/product-is/issues/16372

References :

- [1] Changed UI

> Screenshot 2023-10-31 at 05 27 02

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/connections/pages/connections.tsx [hunk 1]
```diff
@@ -107,7 +107,7 @@ const ConnectionsPage: FC<ConnectionsPropsInterface> = (props: ConnectionsPropsI
     const [ localAuthenticatorList, setLocalAuthenticatorList ] = useState<AuthenticatorInterface[]>([]);
     const [ filter, setFilter ] = useState<string>(null);
     const [ appendConnections, setAppendConnections ] = useState<boolean>(false);
-    const [ isPaginating, setIsPaginating ] = useState<boolean>(false);
+    const [ isPaginating ] = useState<boolean>(false);
```
Comment (Achintha444) [inline apps/console/src/features/connections/pages/connections.tsx:110]: If we are not setting this state variable, we can remove it

File: identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties [hunk 2]
```diff
@@ -431,6 +431,7 @@ fido.proceed=Proceed
 fido.cancel=Cancel
 fido.submit=Submit
 fido.authenticator=Passkey
+provisioned.user.not.found=No associated local mapped user was found.
```
Comment (pavinduLakshan) [inline identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties:434]: Do we need the phrase "local mapped" here? IMO, it'd be too technical for an end user.

We sometimes ago removed a similar error message by replacing it with more user friendly error msg. Therefore, it'd be better if we can review this.

File: apps/console/src/features/connections/pages/connections.tsx (hunk 1 above)
Comment (AfraHussaindeen) [reply apps/console/src/features/connections/pages/connections.tsx]: Resolved via https://github.com/wso2/identity-apps/pull/4460/commits/87e1442cda8f268962da73384711efa2a5daa023

[the remark this answers is not in the document]
Comment (AfraHussaindeen) [reply apps/console/src/features/connections/pages/connections.tsx]: Resolved via https://github.com/wso2/identity-apps/pull/4460/commits/48c61090e622c9e7dc5f6f1492458c9a51473138

File: identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties (hunk 2 above)
Comment (AfraHussaindeen) [reply identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties]: Resolved via https://github.com/wso2/identity-apps/pull/4460/commits/90354fddbe665b4b6574fe0a093710dade786d46

**Review by Achintha444 — APPROVED** [review]

**Review by Achintha444 — APPROVED** [review]

**Review by pavinduLakshan — DISMISSED** [review]

File: .changeset/sweet-crabs-smile.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+"@wso2is/access-control": patch
+"@wso2is/dynamic-forms": patch
+"@wso2is/identity-apps-core": patch
+"@wso2is/myaccount": patch
+"@wso2is/common": patch
+"@wso2is/theme": patch
+"@wso2is/console": patch
+"@wso2is/core": patch
+"@wso2is/form": patch
+"@wso2is/i18n": patch
```
Comment (pavinduLakshan) [inline .changeset/sweet-crabs-smile.md:12]: We should only select the packages updated with this PR. Please generate the changeset again only with the changed packages. :)

**Review by pavinduLakshan — CHANGES_REQUESTED** [review]

File: .changeset/sweet-crabs-smile.md (hunk 3 above)
Comment (AfraHussaindeen) [reply .changeset/sweet-crabs-smile.md:12]: Resolved via https://github.com/wso2/identity-apps/pull/4460/commits/1e515a6da610b6955d5d3b0ea4cc5413e7e640b3

File: .changeset/sweet-crabs-smile.md
Comment (pavinduLakshan) [inline .changeset/sweet-crabs-smile.md:1]: Let's delete this file

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**